lib/customer.rb

```diff
@@ -0,0 +1,37 @@
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
+
+  def initialize(id, email, address)
+    @id = id
+    @email = email
+    @address = address
+  end
+
+  def self.all
+    @customers = []
```

Comment:
Should we be using the instance variable `@customers` here? I'm not convinced that we should -- let's talk about this if you have questions